controller/Delivery.php

```diff
@@ -0,0 +1,432 @@
+        $deliveries = $this->getDeliveries($testCenter);
+
+        $this->composeView(
+            'deliveries-listing',
+            array(
+                'list' => $deliveries
+            ),
+            array(
+            Breadcrumbs::testCenters(),
+            Breadcrumbs::testCenter($testCenter, $this->getTestCenters()),
+            Breadcrumbs::deliveries($testCenter,
+                array(
+                Breadcrumbs::diagnostics($testCenter),
+                Breadcrumbs::reporting($testCenter)
+            ))
+        ));
+    }
+
+    /**
+     * Display all delivery executions of the selected delivery and test center
+     */
+    public function monitoring()
+    {
+
+        $testCenter    = $this->getCurrentTestCenter();
+        $delivery      = $this->getCurrentDelivery();
+        $executionData = $this->getDeliveryExecutions($delivery);
+
+        $this->composeView(
+            'delivery-manager',
+            array(
+                'id' => $delivery->getUri(), //change key to delivery for better consistency
+                'testSite' => $testCenter->getUri(), //change key to delivery for better consistency
+                'set' => $executionData //change this to list for better consistency
+            ),
+            array(
+                Breadcrumbs::testCenters(),
+                Breadcrumbs::testCenter($testCenter, $this->getTestCenters()),
+                Breadcrumbs::deliveries($testCenter,
+                    array(
+                        Breadcrumbs::diagnostics($testCenter),
+                        Breadcrumbs::reporting($testCenter)
+                    )
+                ),
+                Breadcrumbs::deliveryMonitoring($testCenter, $delivery, $this->getDeliveries())
+            )
+        );
+    }
+
+    /**
+     * List available test takers to assign to a delivery
+     *
+     * @throws \Exception
+     * @throws \common_Exception
+     * @throws \oat\oatbox\service\ServiceNotFoundException
+     */
+    public function testTakers() {
+
+        $delivery = $this->getCurrentDelivery();
+        $testCenter = $this->getCurrentTestCenter();
+        
+        try {
+
+            $deliveryService = $this->getServiceManager()->get('taoProctoring/delivery');
+            $currentUser = \common_session_SessionManager::getSession()->getUser();
+
+            $requestOptions = $this->getRequestOptions();
+            $users = $deliveryService->getAvailableTestTakers($currentUser, $delivery->getUri(), $requestOptions);
+            $testTakers = $this->getTestTakersPage($users, $requestOptions);
+
+            $this->composeView(
+                'assign-test-takers',
+                array(
+                    'id' => $delivery->getUri(),
+                    'title' =>  __('Assign test takers to %s', $delivery->getLabel()),
+                    'testsite' => $testCenter->getUri(),
+                    'set' => $testTakers //change it to list for consistency
```

Comment:
The pagination is included in the refactored version of getTestTakerPage() so I guess there is no need there anymore ?